src/app/shell/personal-cabinet/provider/create-provider/create-photo-form/create-photo-form.component.ts

```diff
@@ -26,6 +29,9 @@ export class CreatePhotoFormComponent implements OnInit {
     Validators.minLength(ValidationConstants.INPUT_LENGTH_3),
     Validators.maxLength(ValidationConstants.MAX_DESCRIPTION_LENGTH_2000)
   ]);
+  SectionItemsFormArray = new FormArray([]);
+  isPristine = true;
```

Comment:
remove this value as well

Reply:
we need SectionItemsFormArray for the view template

Reply:
isPristine is not needed